frontend/src/views/Files.vue

```diff
@@ -112,8 +112,10 @@ export default {
   },
   watch: {
     '$route': 'fetchData',
-    'reload': function () {
-      this.fetchData()
+    'reload': function (newV) {
+       if (newV.isReset === false) {
+        this.fetchData()
+      }
```

Comment:
We can control the reload with a single boolean.

```suggestion
    'reload': function (value) {
       if (value) {
        this.fetchData()
      }
```

Reply:
yes,keep it simple.thanks